exercises/word-search/word_search_test.py

```diff
@@ -36,48 +137,179 @@ def test_vertical_words_top_to_bottom(self):
             self.example.search('ecmascript'),
             (Point(9, 0), Point(9, 9))
         )
+        self.assertEqual(
```

Comment:
There are a lot of redundant and misplaced assertions. Please check that each assertion actually belongs under its test method (Example: checks for "clojure" in `self.example` should only appear in tests for horizontal words).